package.json

```diff
@@ -56,6 +56,10 @@
       "prettier --write",
       "eslint --fix"
     ],
+    "*.tsx": [
```

Comment:
I think we can just combine this with the above `*.ts` pattern to make it `*.{ts,tsx}`. Maybe even `*.tsx?` works?

Reply:
I tried `*.tsx?` previously and it wouldn't work, but `*.{ts,tsx}` might do the trick.

Reply:
Yeah, I noticed the docs have `*.{ts,tsx}` (or actually I think it was `*.{js,jsx}`) so at least that much should work. Not sure why `*.tsx?` doesn't work but I suppose it's due to the glob library they're using.